juanevillam/xtrovert
# Implement Responsive Navbar Component

## Overview  
This PR implements a responsive Navbar component, enhancing navigation and user experience across both desktop and mobile views.

---

## Key Changes  

### **Features**  
- Added Navbar links for primary navigation.  
- Implemented Navbar dropdowns for additional options.  
- Included a tooltip component to enhance user experience on hover.  

### **Enhancements**  
- Added scroll animation for mobile view to improve user interaction.  
- Ensured the Navbar is responsive and displays correctly on various screen sizes.  

---

## Reasons for Changes  
Introducing a fully functional and styled Navbar improves the application's navigation, making it more intuitive and user-friendly. The responsiveness ensures a consistent experience across devices.

---

## How to Test  
1. Run the application and navigate through different pages.  
2. Verify that the Navbar displays correctly on desktop and mobile devices.  
3. Test the dropdown menus to ensure they function properly.  
4. Hover over Navbar links and dropdowns on desktop to check the tooltip functionality.  
5. On mobile devices, test the scroll animation to ensure it enhances usability.  
6. Check for any responsiveness issues on various screen sizes.
